core/__init__.py

```diff
@@ -398,7 +398,7 @@ def configure_general():
     FFMPEG_PATH = CFG['General']['ffmpeg_path']
     SYS_PATH = CFG['General']['sys_path']
     CHECK_MEDIA = int(CFG['General']['check_media'])
-    REQUIRE_LAN = CFG['General']['require_lan'] or None
+    REQUIRE_LAN = CFG['General']['require_lan'].split(',') or None
```

Comment:
A split of an empty string will return a list of [''].

Reply:
Thanks for identifying this... I was sure I had done a test with this set to an empty string... but clearly this was missed.
I have just submitted a new PR to Nightly now. Are you able to confirm this still works for empty string?